test.js

```diff
@@ -50,18 +50,18 @@ test('doesn\'t add unnecessary escape codes', t => {
 	t.is(sliceAnsi('\u001B[31municorn\u001B[39m', 0, 3), '\u001B[31muni\u001B[39m');
 });
 
-test.failing('can slice a normal character before a colored character', t => {
+test('can slice a normal character before a colored character', t => {
 	t.is(sliceAnsi('a\u001B[31mb\u001B[39m', 0, 1), 'a');
 });
 
-test.failing('can slice a normal character after a colored character', t => {
-	t.is(sliceAnsi('\u001B[31ma\u001B[39mb', 1, 2), 'b');
+test('can slice a normal character after a colored character', t => {
+	t.is(sliceAnsi('\u001B[31ma\u001B[39mb', 1, 2), '\u001B[31m\u001B[39mb');
 });
 
 // See https://github.com/chalk/slice-ansi/issues/22
-test.failing('can slice a string styled with both background and foreground', t => {
+test('can slice a string styled with both background and foreground', t => {
 	// Test string: `chalk.bgGreen.black('test');`
-	t.is(sliceAnsi('\u001B[42m\u001B[30mtest\u001B[39m\u001B[49m', 0, 1), '\u001B[42m\u001B[30mt\u001B[39m\u001B[49m');
+	t.is(sliceAnsi('\u001B[42m\u001B[30mtest\u001B[39m\u001B[49m', 0, 1), '\u001B[42m\u001B[30mt\u001B[49m\u001B[39m');
```

Comment:
The order of end escapes should be preserved.

Reply:
:+1: